src/Tribe/RSVP/Early_Access/Template.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tribe\Tickets\RSVP\Early_Access;
```

Comment:
We should avoid calling it Early Access because that will be the final name.
Maybe call it `V2` or `Unified`, not sure. The Green Devs should agree on a final name since we want to avoid deprecations.